File: Java/AutomationFramework/src/com/app/UITests/LoginPageTests/UserLoginTest.java
package com.app.UITests.LoginPageTests;

import com.app.pages.LoginPage;
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class UserLoginTest extends BaseLoginTest{
    LoginPage Login_Page;

    @BeforeMethod
    public void SetUp() {
        Login_Page = LoginPage.GetLoginPage()
                .Init_Elements();
    }

    @Test
    public void UserTest()
    {
        String username = GetProperty("username");
        String password = GetProperty("password");
        boolean isLogoPresent = Login_Page.EnterUsername(username)
            .EnterPassword(password)
            .ClickLogin()
            .IsLogoPresent();

        Assert.assertEquals(isLogoPresent, true);
    }
}

File: DotNet/Framework/AutomationFramework/Tests.UITests/LoginPageTests/UserLoginTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.UITests.LoginPageTests
{
    [Parallelizable]
    [TestFixture]
    public class UserLoginTest : BaseTestFixtures
    {
        protected string Username;
        protected string Password;
        LoginPage Login_Page;

        [SetUp] 
        public void SetUp()
        {
            Login_Page = LoginPage.Login_Page;
            Username = GetConfigurationValue("Username");
            Password = GetConfigurationValue("Password");
        }

        [Test]
        public void UserLogin()
        {
            Login_Page.EnterUsername(Username)
                .EnterPassword(Password)
                .ClickLogin()
                .IsLogoPresent(out bool isLogoPresent);

            Assert.That(isLogoPresent, Is.EqualTo(true));
        }
    }
}
